tests/test_cases/test_cocotb/test_tests.py

```diff
@@ -62,14 +62,11 @@ async def test_timeout_testdec_pass(dut):
     await Timer(10, "ns")
 
 
-# VCS fails gh-4326
-@cocotb.test(
-    timeout_time=10, timeout_unit="ns", expect_fail="vcs" in cocotb.SIM_NAME.lower()
-)
+@cocotb.test(timeout_time=10, timeout_unit="ns")
 async def test_timeout_testdec_simultaneous(dut):
     try:
         await cocotb.triggers.with_timeout(
-            Timer(1, "ns"), timeout_time=1, timeout_unit="ns"
+            Timer(2, "ns"), timeout_time=1, timeout_unit="ns"
```

Comment:
I'm not sure what exactly this is even testing anymore. You can yeet this test, but I suppose this is fine.

Reply:
I guess it test that `with_timeout` works as expected?

Reply:
That's in test_timing_triggers too I think.

Reply:
Yes.